ob_intern/Custom View/Hashtag/SocialPostHashTagView.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  SocialPostHashTagView.swift
+//  Joy
+//
+//  Created by TanGMo SuChaYA on 24/5/22.
+//  Copyright © 2022 ookbee. All rights reserved.
+//
+
+import UIKit
+
+protocol SocialPostHashTagViewDelegate: AnyObject {
+    func setHeightConstraint(viewHeight : CGFloat)
+    func setViewHidden(isHidden: Bool)
+}
+
+class SocialPostHashTagView: InitializeXibView {
+
+    @IBOutlet weak var collectionView: UICollectionView!
+    @IBOutlet weak var collectionHeightConstraint: NSLayoutConstraint!
+    
+    weak var delegate: SocialPostHashTagViewDelegate?
+    
+    var hashtags : [String] = []
+    var viewIsHidden: Bool = true
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        self.setupView()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    func setupView() {
+        collectionView.dataSource = self
+        collectionView.delegate = self
+        collectionView.register(cell: DisplayHashTagCollectionViewCell.self)
+        collectionView.reloadData()
+        
+        let layout = LeftAlignedFlowLayout()
+        layout.scrollDirection = .horizontal
+        layout.sectionInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 10)
+        self.collectionView.setCollectionViewLayout(layout, animated: true)
+        
+    }
+    
+    func setData(hashtags: [String]) {
+        self.hashtags = hashtags
+        
+        collectionView.reloadData()
```

Comment:
ทำให้ไปในทางเดียวกันนะ ถ้าจะใส่ self ก็ใส่  หรือไม่ใส่ก็ไม่ใส่ให้หมด 
```suggestion
        self.collectionView.reloadData()
```